hooks/post/30-upgrade.rb

```diff
@@ -166,6 +170,7 @@ def fail_and_exit(message)
       upgrade_step :import_distributions, :long_running => true
       upgrade_step :import_puppet_modules, :long_running => true
       upgrade_step :import_subscriptions, :long_running => true
+      upgrade_step :import_product_content, :long_running => true, :run_always => true
```

Comment:
Looks fine, just 1 comment. Will this need to be kept around in the future for say 3.7 or is this just a 1 time thing for 3.6? The reason I ask is about the run_always